test/models/friend_request_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
```

Comment:
Please add test for associations here.. You may use shoulda-gem I mentioned in my comment on friendship_test